pkg/apis/ptp/v1/ptpoperatorconfig_types.go

```diff
@@ -24,6 +25,21 @@ type PtpOperatorConfigStatus struct {
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
 }
 
+// PtpEventConfig defines the desired state of event framework
+// +k8s:openapi-gen=true
+type PtpEventConfig struct {
+	//EnableEventPublisher will deploy event proxy as a sidecar
+	EnableEventPublisher  bool `json:"enableEventPublisher,omitempty"`
+	//clock state to stay in holdover state in secs
```

Comment:
lets add to the description the default time we use if the user doesn't provide one

Reply:
Added default to decriptioin